identity-did/src/verification/method_type.rs

```diff
@@ -11,16 +11,20 @@ use crate::error::Result;
 /// Supported verification method types.
 #[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord, Deserialize, Serialize)]
 #[repr(u32)]
+// TODO: use string representations to avoid breaking changes from reordering.
```

Comment:
Aren't we already using string representations? In any case, I think we could remove the `#[repr(u32]` and `as_u32` and `from_u32` methods after #729, as it's only used in the current `KeyLocation`.

Reply:
For `serde` yes we still use string representations since `repr(u32)` alone doesn't change the serialization. But `wasm-bindgen` uses numeric representations for flat enums in the Wasm bindings which is another thing I'm concerned about breaking (I don't think there's really a way around that though).

I think I'll go ahead and remove the `repr(u32)` and associated methods at least.

Reply:
Removed the `u32` representation and methods.